test/cli/detailed-errors/test.out

```diff
@@ -0,0 +1,168 @@
+shapes.rb:10: Expected `{a: Integer, b: String}` but found `{}` for argument `x` https://srb.help/7002
+    10 |takes_shape({})
+                    ^^
+  Expected `{a: Integer, b: String}` for argument `x` of method `Object#takes_shape`:
+    shapes.rb:5:
+     5 |sig { params(x: {a: Integer, b: String}).void }
+                     ^
+  Got `{} (shape of T::Hash[T.untyped, T.untyped])` originating from:
+    shapes.rb:10:
+    10 |takes_shape({})
+                    ^^
+
+shapes.rb:11: Expected `{a: Integer, b: String}` but found `{a: Integer(1)}` for argument `x` https://srb.help/7002
+    11 |takes_shape({a: 1})
+                    ^^^^^^
+  Expected `{a: Integer, b: String}` for argument `x` of method `Object#takes_shape`:
+    shapes.rb:5:
+     5 |sig { params(x: {a: Integer, b: String}).void }
+                     ^
+  Got `{a: Integer(1)} (shape of T::Hash[T.untyped, T.untyped])` originating from:
+    shapes.rb:11:
+    11 |takes_shape({a: 1})
+                    ^^^^^^
+
+shapes.rb:14: Expected `{a: Integer, b: String}` but found `{a: Integer(1), c: Integer(1)}` for argument `x` https://srb.help/7002
+    14 |takes_shape({a: 1, c: 1})
+                    ^^^^^^^^^^^^
+  Expected `{a: Integer, b: String}` for argument `x` of method `Object#takes_shape`:
+    shapes.rb:5:
+     5 |sig { params(x: {a: Integer, b: String}).void }
+                     ^
+  Got `{a: Integer(1), c: Integer(1)} (shape of T::Hash[T.untyped, T.untyped])` originating from:
+    shapes.rb:14:
+    14 |takes_shape({a: 1, c: 1})
+                    ^^^^^^^^^^^^
+
+shapes.rb:15: Expected `{a: Integer, b: String}` but found `{c: Integer(1), d: Integer(1)}` for argument `x` https://srb.help/7002
+    15 |takes_shape({c: 1, d: 1})
+                    ^^^^^^^^^^^^
+  Expected `{a: Integer, b: String}` for argument `x` of method `Object#takes_shape`:
+    shapes.rb:5:
+     5 |sig { params(x: {a: Integer, b: String}).void }
+                     ^
+  Got `{c: Integer(1), d: Integer(1)} (shape of T::Hash[T.untyped, T.untyped])` originating from:
+    shapes.rb:15:
+    15 |takes_shape({c: 1, d: 1})
+                    ^^^^^^^^^^^^
+
+shapes.rb:18: Expected `{a: Integer, b: String}` but found `{a: Integer(1), b: Integer(1)}` for argument `x` https://srb.help/7002
+    18 |takes_shape({a: 1, b: 1})
+                    ^^^^^^^^^^^^
+  Expected `{a: Integer, b: String}` for argument `x` of method `Object#takes_shape`:
+    shapes.rb:5:
+     5 |sig { params(x: {a: Integer, b: String}).void }
+                     ^
+  Got `{a: Integer(1), b: Integer(1)} (shape of T::Hash[T.untyped, T.untyped])` originating from:
+    shapes.rb:18:
+    18 |takes_shape({a: 1, b: 1})
+                    ^^^^^^^^^^^^
+  Possible explanation:
+    `Integer(1)` is not a subtype of `String` for key `Symbol(:b)`
```

Comment:
We probably want to do a better job of pretty-printing these. Ideally it would not say `Symbol(:b)` but rather something like "for key `b:`"